src/realtime_metrics/main.py

```diff
@@ -363,7 +365,7 @@ def availability_acceptable_stop_time_updates(stop_time_updates: list[tuple[Trip
         trip_update = update[0]
 
         # get time in minutes
-        time_in_minutes = int(trip_update.timestamp.replace(tzinfo=timezone.utc).timestamp() / 60)
+        time_in_minutes = int(trip_update.timestamp.replace(tzinfo=timezone.utc).timestamp() / 60) * 60
```

Comment:
/60 * 60? So this are seconds now? Rename the variable please if so.

Reply:
I kept the timestamp in seconds, as timestamps are defined this way. This operations removes the part responsible for individual seconds and thereby round down to the last minute. I renamed the variable and adapted the comment to make this more clear.